hooks/useProfileSearch.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useCallback, useEffect } from 'react'
+import { useAccount } from 'wagmi';
+import { ZoraProfile } from '@/lib/zora.types';
+
+const useProfileSearch = () => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const [profile, setProfile] = useState<ZoraProfile[]>([])
+
+  const { address } = useAccount();
+
+  const fetchResults = useCallback(async (query: string) => {
+    if (query.length > 0) {
+      try {
+        const response = await fetch(`https://api.myco.wtf/api/profile?address=0x6e0248e7909CDc4F3Ea2b381473D07953A9490ed`)
```

Comment:
Why is `address` hard-coded?
```suggestion
        const response = await fetch(`https://api.myco.wtf/api/profile?address=${address}`)
```